src/components/list-archive-all-tags/list-archive-all-tags.js

```diff
@@ -0,0 +1,60 @@
+import * as React from "react"
+
+import "../../scss/styles.scss"
+import "../../js/globals"
+
+import ListItem from "../list-item/list-item"
+import Chip from "../chip/chip"
+import { useStaticQuery, graphql } from "gatsby"
+
+const ListArchiveAllTags = ({
+  background,
+  noSpace,
+}) => {
+
+  const data = useStaticQuery(graphql`
+    query {
+      allContent {
+        group(field: components___hero___kangaroo___tags) {
+          fieldValue
+          totalCount
+        }
+      }
+    }
+  `)
+
+  let container = "container-xxl"
+  let row = "row"
+
+  let styles = 'section-editorial'
+    + `${background ? ' bg-' + background : ''}`
+    + `${noSpace ? ' py-0' : ''}`
+    + `${background === "dark" ? ' text-white' : ''}`
```

Comment:
We should use [`classnames`](https://www.npmjs.com/package/classnames) here. 

In general, the rest of the site doesn't but we can fix that other code already checked in later with a big pass of linting.

Reply:
What do you mean? To use variable name `classNames` instead of `styles`? Or else?

Reply:
`classnames` the package, take a look at the link.

Reply:
Ok. Clear. I would prefer to do it with a whole dedicated refactoring/linting "post-archives". We have this kind of approach in too many files right now.

Reply:
@bfabio just did for the new list-archives components. Check it out. I close this.